plugins/backstage-plugin-coder/src/components/CoderWorkspacesCard/SearchBox.test.tsx

```diff
@@ -28,18 +27,14 @@ async function renderSearchBox(input?: RenderInputs) {
   const { queryFilter = 'owner:me' } = input ?? {};
   const onFilterChange = jest.fn();
 
-  const mockContext: WorkspacesCardContext = {
+  const mockContext: Partial<WorkspacesCardContext> = {
```

Comment:
Felt like adding a bunch of unused values added noise, and made it harder to understand what really mattered in the test case

Reply:
Makes sense, that does clear it up.